File: subset/subset.py
# Starter Code
class Solution:
    def isSubset(self,arr1,arr2,n,m):
        check_subset=-1
        #Write your code here without removing the exising code
	    #the variable 'arr1' and 'arr2' contains array of integers of size n and m respectively.
	    #modified the integer check_subset contain the output of the program.
        return check_subset

if __name__ == '__main__':
        n = int(input())
        m = int(input())
        str1 = input()
        str2 = input()
        arr1 = [int(int(i)) for i in str1.strip('[]').split(',')]
        arr2 = [int(int(i)) for i in str2.strip('[]').split(',')]
        x = Solution()
        ans = x.isSubset(arr1,arr2,n,m)
        print(ans)

File: subset/subset.java

	
	public static void main(String[] args)
    {
		Scanner sc=new Scanner(System.in);
            int n = sc.nextInt(); 
            int m= sc.nextInt(); 
			int []arr1= Arrays.stream(sc.next().split("[,\\[\\]]")).filter(w-> !w.equals("")).mapToInt(Integer::parseInt).toArray();
            int []arr2= Arrays.stream(sc.next().split("[,\\[\\]]")).filter(w-> !w.equals("")).mapToInt(Integer::parseInt).toArray();
		    Solution g=new Solution();
            int ans=g.isSubset(arr1,arr2,n,m);
            System.out.println(ans);
          
	}
}
class Solution
{
	public static int isSubset(int arr1[],int arr2[],int n,int m)
        {
            
			int check_subset =-1;
            //Write your code here without removing the existing code.
	        //the variable 'arr1' and 'arr2' contains array of integers of size n and m respectively.
	        //modified the check_subset contain the output of the program in integer format.
            return check_subset;
        }
}